lib/netconf/netconf.go

```diff
@@ -83,6 +83,7 @@ type Chain struct {
 	PortalAddress string `json:"portal_address"`        // Address of the omni portal contract on the chain
 	DeployHeight  uint64 `json:"deploy_height"`         // Height that the portal contracts were deployed
 	IsOmni        bool   `json:"is_omni,omitempty"`     // Whether this is the Omni chain
+	BlockTime     uint32 `json:"block_time"`            // Block time in milliseconds
```

Comment:
`BlockPeriod`

Reply:
Please, use `time.Duration` instead of `uint32` for the duration field.